src/ItemDelegate.cpp

```diff
@@ -103,25 +103,22 @@ void ItemDelegate::paintLoginItem(QPainter *painter
                 paintArrow(painter, option);
             else
                 paintFavorite(painter, option, pLoginItem->favorite());
+
+            QPen pen;
+            pen.setColor(QColor("#3D96AF"));
+            painter->setPen(pen);
+            QFont font = qApp->font();
+            font.setBold(true);
+            font.setItalic(true);
+            font.setPointSize(10);
+            painter->setFont(font);
+
+            QString indent(8, ' ');
+            painter->drawText(option.rect, Qt::AlignVCenter, indent + pLoginItem->name());
```

Comment:
Adding spaces here instead of the model is not what I commented...
Here you need to draw the text by moving the `option.rect` and not by adding spaces...

Reply:
Ok.